push_to_docker.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -eu.
```

Comment:
This isn't meant to end in the dot, right?

Reply:
You are right, gonna ✂️ it.